scripts/books_script.php

```diff
@@ -0,0 +1,144 @@
+<?php
```

Comment:
Please try to move everything into a class and keep the books_script.php file clean.
We should keep the initial validation for arguments in this file, but the code related to API request/processing of data should stay in the methods of a new class.